enzy_htp/structure/atom.py

```diff
@@ -128,7 +128,16 @@ def element(self):
                 return self.parent.element
             else:
                 # case: in ligand atoms are named like this H1
-                return re.match("^[A-Z][a-z]?", self.name).group()
+                for ch in self.name:
+                    if not ch.isnumeric():
+                        return ch
```

Comment:
Oh I see there are names starting with number. But the current logic also dont work for atoms like Mg or Cl, which will be M and C returned

Reply:
Good point. Replaced the logic and updated it in a commit to not be susceptible to the bug.

Reply:
Nice change!

But the current logic still dont work with atom names like CH3 in e.g. FAH ligand.

Reply:
I also found in a ligand called DCE, Cl's name is CL1, CL2 which is kind hard to deduce using just the name. The ultimate solution would be using a mapper that record all these.

Do you mind if I push some unit tests and changes related to this on this branch?

Reply:
I fixed it and filed a TODO for the CL case. Also added some unit tests